pdns/dnsrecords.cc

```diff
@@ -489,6 +489,227 @@ string EUI64RecordContent::getZoneRepresentation(bool noDot) const
 
 /* EUI64 end */
 
+/* APL start */
+/* https://tools.ietf.org/html/rfc3123 */
+void APLRecordContent::report(void)
+{
+  regist(1, QType::APL, &make, &make, "APL");
+}
+
+// Parse incoming packets (e.g. nsupdate)
+std::shared_ptr<DNSRecordContent> APLRecordContent::make(const DNSRecord &dr, PacketReader& pr) {
+  uint8_t temp;
+  APLRDataElement ard;
+  int processed = 0;
```

Comment:
Let's prevent a possible issue when `sizeof(int) == 2`.

```suggestion
  size_t processed = 0;
```